src/adjoint/adjointAPI.F90

```diff
@@ -815,8 +815,7 @@ subroutine solveAdjoint(RHS, psi, checkSolution, nState)
        ! Determine the maximum time using MPI reduce
        ! with operation mpi_max.
 
-       ! call mpi_reduce(timeAdjLocal, timeAdj, 1, adflow_real, &
-       !      mpi_max, 0, ADFLOW_COMM_WORLD, ierr)
+       call mpi_reduce(timeAdjLocal, timeAdj, 1, adflow_real, mpi_max, 0, ADFLOW_COMM_WORLD, ierr)
```

Comment:
I wonder why this call was removed. I know Sandy was doing some timing studies during the ADflow paper, maybe happened around then?

Reply:
It was commented out in [this commit](https://github.com/mdolab/adflow/commit/aa69a61ee47dbb2d19f312f6f54904b1fd1dfd29), probably just for debugging and then left like that by mistake.